.github/renovate.json5

```
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "schedule:daily",
+    ":dependencyDashboard",
+    ":semanticCommits",
+    ":semanticCommitScope(deps)",
+    "group:linters",
+    "group:googleapis",
+    "group:goOpenapi"
+  ],
+  "packageRules": [
+    {
+      "datasources": ["docker"],
+      "updateTypes": ["patch"],
+      "groupName": "docker patch updates"
+    },
+    {
+      "datasources": ["go"],
+      "updateTypes": ["patch", "minor"],
+      "groupName": "go.mod non-major updates"
+    },
+    {
+      "datasources": ["npm"],
+      "updateTypes": ["patch", "minor"],
+      "groupName": "npm non-major updates"
```

```suggestion
      "groupName": "npm dependencies"
```

This should make the PR title better to understand